### ISSUE TITLE
Add favourite support

### ISSUE THREAD

**Issue by mmatyas** [mmatyas/pegasus-frontend#44]
- [x] implement an API
- [ ] add support for it in the default theme
  - [ ] indicator
  - [ ] toggle

**Comment by mmatyas** [comment]
This depends on #119 (because we need a heart icon or something) and #14/#11 (since we have to store this setting somewhere).

**Comment by AndersHP84** [comment]
In my opinion: 

Need to have: Favorites with no heart/star icon.
Nice to have: The Heart/Star icon is visible in general, but _hidden_ if the "favorites" toggle is "on".

**Comment by AndersHP84** [comment]
By the way: latest EmulationStation lets you toggle favorite on/off for a game by hitting Y. 
Couldn't this function be copied directly into Pegasus?

**Comment by mmatyas** [comment]
Sure, it's not a problem. I'll just need to finish #14 first to actually store the setting.
